exercises/operations/operations2.cairo

```diff
@@ -0,0 +1,25 @@
+// Mary is buying apples. The price of an apple is calculated as follows:
+// - An apple costs 3 rustbucks.
```

Comment:
maybe rename to cairobuck? 

also we should probably move this to exercises/quiz1 as it's a summary of the previous sections, not really related to operators